inference-engine/tests/functional/plugin/shared/include/behavior/behavior_test_plugin.hpp

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
as far as I can see, all files that are moved already, don't have `behavior` prefix. Why this is an exception?

Reply:
I repeated the writing from the old tests, this is not the final solution.